scene/resources/packed_scene.cpp

```diff
@@ -730,8 +819,23 @@ Error SceneState::_parse_node(Node *p_owner, Node *p_node, int p_parent_idx, Has
 		if (!pinned_props.has(name)) {
 			bool is_valid_default = false;
 			Variant default_value = PropertyUtils::get_property_default_value(p_node, name, &is_valid_default, &states_stack, true);
+
 			if (is_valid_default && !PropertyUtils::is_property_value_different(value, default_value)) {
-				continue;
+				// FIXME: This saves local_to_scene resources NO MATTER WHAT, even if their value hasn't changed.
+				// This can cause .tscn files to get larger than needed, so the default value check should take into account
+				// local resources somehow.
```

Comment:
This is expected tbh. If a resource is local to scene, it gets duplicated, which makes the value modified.